schemas/config.json

```diff
@@ -0,0 +1,367 @@
+                    },
+                    "required": ["address"]
+                }
+            ]
+        }
+    },
+    "type": "object",
+    "properties": {
+        "systems": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "name": {"$ref": "#/defs/alphanum_string"},
+                    "descr": {"type": "string"},
+                    "hostnames": {
+                        "type": "array",
+                        "items": {"type": "string"}
+                    },
+                    "modules_system": {
+                        "type": "string",
+                        "enum": ["tmod", "tmod31", "tmod32", "tmod4", "lmod"]
+                    },
+                    "modules": {"$ref": "#/defs/modules_list"},
+                    "variables": {"$ref": "#/defs/envvar_list"},
+                    "prefix": {"type": "string"},
+                    "stagedir": {"type": "string"},
+                    "outputdir": {"type": "string"},
+                    "perflogdir": {"type": "string"},
+                    "resourcesdir": {"type": "string"},
+                    "partitions": {
+                        "type": "array",
+                        "items": {
+                            "type": "object",
+                            "properties": {
+                                "name": {"$ref": "#/defs/alphanum_string"},
+                                "descr": {"type": "string"},
+                                "scheduler": {
+                                    "type": "string",
+                                    "enum": ["local", "pbs", "slurm", "squeue"]
+                                },
+                                "launcher": {
+                                    "type": "string",
+                                    "enum": [
+                                        "alps",  "ibrun", "local", "mpirun",
+                                        "mpiexec", "srun", "srunalloc", "ssh"
+                                    ]
+                                },
+                                "access": {
+                                    "type": "array",
+                                    "items": {"type": "string"}
+                                },
+                                "environs": {
+                                    "type": "array",
+                                    "items": {"type": "string"}
+                                },
+                                "container_platforms": {
+                                    "type": "array",
+                                    "items": {
+                                        "type": "object",
+                                        "properties": {
+                                            "name": {
+                                                "type": "string",
+                                                "enum": ["Docker", "Sarus",
```

Comment:
why are we removing shifter?

Reply:
It is according to what we have in the documentation.